### PR TITLE
zh-translation:/content/zh/faq/security/secure-ingress.md

### PR DESCRIPTION
**Pull request by qunqiang**
ref: https://github.com/servicemesher/istio-official-translation/issues/1429

### CODE REVIEW

File: content/zh/faq/security/secure-ingress.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
-title: How to configure Istio Ingress to only accept TLS traffic?
+title: Istio 中如何配置 Ingress 使其仅处理 TLS 连接？
 weight: 130
 ---
 
-By following the instructions in the
-[Secure Ingress Traffic](/docs/tasks/traffic-management/ingress/secure-ingress-mount) task,
-Istio Ingress can be secured to only accept TLS traffic.
+依照[安全的网关(基于文件挂载的方式)](/docs/tasks/traffic-management/ingress/secure-ingress-mount) 任务中的描述进行配置，能够让 Istio Ingress 只处理 TLS 连接。
```
Comment (rootsongjc) [inline content/zh/faq/security/secure-ingress.md]: 请使用中文的括号。

**Comment by rootsongjc** [comment]
@qunqiang Please sign the CLA first.

File: content/zh/faq/security/secure-ingress.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
 ---
-title: How to configure Istio Ingress to only accept TLS traffic?
+title: Istio 中如何配置 Ingress 使其仅处理 TLS 连接？
 weight: 130
 ---
 
-By following the instructions in the
-[Secure Ingress Traffic](/docs/tasks/traffic-management/ingress/secure-ingress-mount) task,
-Istio Ingress can be secured to only accept TLS traffic.
+依照[安全的网关（基于文件挂载的方式）](/docs/tasks/traffic-management/ingress/secure-ingress-mount) 任务中的描述进行配置，能够让 Istio Ingress 只处理 TLS 连接。
```
Comment (rootsongjc) [inline content/zh/faq/security/secure-ingress.md]: 链接前要加中文前缀 `/zh`。

Comment (qunqiang) [reply content/zh/faq/security/secure-ingress.md]: 哦了， 基本上踩了一遍坑。

**Comment by qunqiang** [comment]
@googlebot I fixed it.

**Pull request merged**